db/migrate/20230113203436_fixup_images.rb

```diff
@@ -0,0 +1,14 @@
+class FixupImages < ActiveRecord::Migration[7.0]
+  def change
+    %i[episode_images feed_images itunes_images].each do |tbl|
+      add_column tbl, :alt_text, :string
+      add_column tbl, :caption, :string
+      add_column tbl, :credit, :string
```

Comment:
Per designs, I'm proactively adding this metadata.  Even though there's nowhere to put it in the RSS yes.